index.html

```diff
@@ -0,0 +1,80 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <script src="src/frame.js"></script>
+    <script src="src/bowling.js"></script>
+    <script src="src/jquery.js"></script>
+    <script src="src/interface.js"></script>
+
+    <link rel="stylesheet" href="public/css/index.css">
+
+    <title>Bowling Scorecard</title>
+</head>
+<body>
+  <h1>Bowling Scorecard</h1>
+  <button id="button0" type="button" value="0">0</button>
+  <button id="button1" type="button" value="1">1</button>
+  <button id="button2" type="button" value="2">2</button>
+  <button id="button3" type="button" value="3">3</button>
+  <button id="button4" type="button" value="4">4</button>
+  <button id="button5" type="button" value="5">5</button>
+  <button id="button6" type="button" value="6">6</button>
+  <button id="button7" type="button" value="7">7</button>
+  <button id="button8" type="button" value="8">8</button>
+  <button id="button9" type="button" value="9">9</button>
+  <button id="button10" type="button" value="10">10</button>
+</body>
+<h2 id="frame0">Frame 1:</h2>
+<div>
+  <span id="0roll1">Roll 1:</span> <span id="0roll2">Roll 2:</span>
+</div>
+
+<h2 id="frame1">Frame 2:</h2>
+<div>
+  <span id="1roll1">Roll 1:</span> <span id="1roll2">Roll 2:</span>
+</div>
+
+<h2 id="frame2">Frame 3:</h2>
+<div>
+  <span id="2roll1">Roll 1:</span> <span id="2roll2">Roll 2:</span>
+</div>
+
+<h2 id="frame3">Frame 4:</h2>
+<div>
+  <span id="3roll1">Roll 1:</span> <span id="3roll2">Roll 2:</span>
+</div>
+
+<h2 id="frame4">Frame 5:</h2>
+<div>
+  <span id="4roll1">Roll 1:</span> <span id="4roll2">Roll 2:</span>
+</div>
+
+<h2 id="frame5">Frame 6:</h2>
+<div>
+  <span id="5roll1">Roll 1:</span> <span id="5roll2">Roll 2:</span>
+</div>
+
+<h2 id="frame6">Frame 7:</h2>
+<div>
+  <span id="6roll1">Roll 1:</span> <span id="6roll2">Roll 2:</span>
+</div>
+
+<h2 id="frame7">Frame 8:</h2>
+<div>
+  <span id="7roll1">Roll 1:</span> <span id="7roll2">Roll 2:</span>
+</div>
+
+<h2 id="frame8">Frame 9:</h2>
+<div>
+  <span id="8roll1">Roll 1:</span> <span id="8roll2">Roll 2:</span>
+</div>
+
+<h2 id="frame9">Frame 10:</h2>
+<div>
+  <span id="9roll1">Roll 1:</span> <span id="9roll2">Roll 2:</span> <span id="9roll3">Roll 3:</span>
+</div>
+  
+</html>
```

Comment:
I like the web interface layout. Having buttons instead of user input is a nice touch as well and i like the idea of graying out buttons to limit user input and remove edge cases.